modules/searchbar/types.ts

```diff
@@ -1,14 +1,22 @@
+import {StyleProp, ViewStyle} from 'react-native'
+
 export type Props = {
 	getRef?: any
 	active?: boolean
 	backButtonAndroid?: 'search' | boolean
 	backgroundColor?: string
-	onCancel: () => any
-	onChange: (data: string) => any
-	onFocus: () => any
-	onSubmit: () => any
+	onCancel: () => unknown
+	onChange: (data: string) => unknown
+	onFocus: () => unknown
+	onSubmit: () => unknown
```

Comment:
Oh by the way. I know I asked for these to be "unknown" the other day, but I've changed my mind (sorry 😅) - I read a post that mentioned that by accepting any return value, you're not using the type checker to its full power, because you may be accidentally returning something. 

So let's switch to "void", and if we need to pass in an async function, we'll cross that bridge when we come to it!